algorithms/baselines/curious_ppo/test_curious_ppo.py

```diff
@@ -0,0 +1,201 @@
+import gc
+import os
+import shutil
+import time
+from os.path import join
+from unittest import TestCase
+
+import numpy as np
+import tensorflow as tf
+
+from algorithms.agent import TrainStatus
+from algorithms.baselines.curious_ppo.agent_curious_ppo import AgentCuriousPPO, CuriousPPOBuffer
+from algorithms.baselines.curious_ppo.enjoy_curious_ppo import enjoy
+from algorithms.baselines.curious_ppo.curious_ppo_utils import parse_args_curious_ppo
+from algorithms.baselines.curious_ppo.train_curious_ppo import train
+from algorithms.baselines.ppo.agent_ppo import ActorCritic
+from algorithms.tests.test_wrappers import TEST_ENV_NAME
+from algorithms.tf_utils import placeholder_from_space, placeholders
+from utils.envs.doom.doom_utils import make_doom_env, doom_env_by_name
+from utils.utils import log, AttrDict, experiments_dir
+
+
+class TestPPO(TestCase):
+    def curious_ppo_train_run(self, env_name=None):
+        test_dir_name = self.__class__.__name__
+
+        args, params = parse_args_curious_ppo(AgentCuriousPPO.Params)
+        if env_name is not None:
+            args.env = env_name
+        params.experiments_root = test_dir_name
+        params.num_envs = 16
+        params.train_for_steps = 60
+        params.initial_save_rate = 20
+        params.batch_size = 32
+        params.ppo_epochs = 2
+        status = train(params, args.env)
+        self.assertEqual(status, TrainStatus.SUCCESS)
+
+        root_dir = params.experiment_dir()
+        self.assertTrue(os.path.isdir(root_dir))
+
+        enjoy(params, args.env, max_num_episodes=1, max_num_frames=50, fps=1000)
+        shutil.rmtree(join(experiments_dir(), params.experiments_root))
+
+        self.assertFalse(os.path.isdir(root_dir))
+
+    def test_curious_ppo_train_run(self):
+        self.curious_ppo_train_run()
+
+    def test_curious_ppo_train_run_goal(self):
+        self.curious_ppo_train_run(env_name='doom_maze_goal')
+
+    def test_buffer_batches(self):
```

Comment:
All the tests except the two tests above are exactly the same as for regular PPO, I don't think we need them. It takes time to run these tests, so I'd rather remove them. Please keep only test_curious_ppo_train_run and test_curious_ppo_train_run_goal